routes/apiRoutes.js

```diff
@@ -1,24 +1,29 @@
 var db = require("../models");
 
 module.exports = function(app) {
-  // Get all examples
-  app.get("/api/examples", function(req, res) {
-    db.Example.findAll({}).then(function(dbExamples) {
-      res.json(dbExamples);
+  // Get all products
+  app.get("/api/getProductList", function(req, res) {
```

Comment:
Using verbs for route path is against the standard practice?

https://blog.mwaysolutions.com/2014/06/05/10-best-practices-for-better-restful-api/

Reply:
Left it unchanged for now. Personally I prefer the explicit name. One more thing to discuss on Saturday